src/main/kotlin/lotto/ui/view/InputView.kt

```diff
@@ -2,24 +2,36 @@ package lotto.ui.view
 
 import lotto.domain.LottoNumber
 
-object InputView {
-    fun getPurchasePrice(): Int {
-        println("구입금액을 입력해 주세요.")
+fun getPurchasePrice(): Int {
+    println("구입금액을 입력해 주세요.")
 
-        return readLine()?.toIntOrNull() ?: 0
-    }
+    return readLine()?.toIntOrNull() ?: 0
+}
 
-    fun getWinningLottoNumbers(): Set<LottoNumber> {
-        println("지난 주 당첨 번호를 입력해 주세요.")
+fun getManualLottoNumbers(): List<Set<LottoNumber>> {
+    println("수동으로 구매할 로또 수를 입력해 주세요.")
 
-        return readLine()?.split(",")?.map { stringNumber ->
-            LottoNumber.of(stringNumber.toInt())
-        }?.toSet() ?: emptySet()
-    }
+    val repeatCount = readLine()?.toIntOrNull() ?: 0
 
-    fun getBonusLottoNumber(): LottoNumber {
-        println("보너스 볼을 입력해 주세요.")
+    println("수동으로 구매할 번호를 입력해 주세요.")
 
-        return LottoNumber.of(readLine()?.toIntOrNull() ?: -1)
+    return List(repeatCount) {
+        readLine()!!.split(",")
+            .map { LottoNumber.of(it.trim().toInt()) }
+            .toSet()
     }
 }
+
+fun getWinningLottoNumbers(): Set<LottoNumber> {
+    println("지난 주 당첨 번호를 입력해 주세요.")
+
+    return readLine()?.split(",")?.map { stringNumber ->
+        LottoNumber.of(stringNumber.trim().toInt())
+    }?.toSet() ?: emptySet()
```

Comment:
https://kotlinlang.org/docs/coding-conventions.html#wrap-chained-calls
> When wrapping chained calls, put the . character or the ?. operator on the next line, with a single indent.
The first call in the chain usually should have a line break before it, but it's OK to omit it if the code makes more sense that way.

코틀린 코딩 컨벤션에 따르면 `.` 또는 `?.`로 이어지는 연속된 체인 호출의 경우 개행을 해줘야 합니다. 체인의 첫 번째 호출은 보통 앞에 줄 바꿈이 있어야 하지만, 코드가 더 의미가 있다면 생략해도 괜찮습니다.

```suggestion
    return readLine()
        ?.split(",")
        ?.map { stringNumber ->
            LottoNumber.of(stringNumber.trim().toInt())
        }
        ?.toSet() ?: emptySet()
```
저도 처음에는 이렇게 작성하는 것이 어색하다고 느꼈지만 익숙해지니 가독성이 좋아진다는 장점을 발견할 수 있었습니다.

Reply:
넵, 컨벤션은 앞으로 꼭 유의하면서 개발할 수 있도록 노력하겠습니다!!